components/list/list-item-drag-drop-mixin.js

```diff
@@ -37,6 +37,214 @@ const isDragSupported = () => {
 	return typeof el.ondragenter === 'function';
 };
 
+class DragState {
```

Comment:
It's unfortunate to have to bring this up since the main focus is usually the component.  Not a big deal.

Reply:
Yeah, we could turn off the rule for classes if we want them to always go at the end, but I also seems good to leave on